### DISCUSSION TITLE
Do response queries work?

### DISCUSSION THREAD

**Discussion by Josh-V-PAL** [usebruno/bruno#915] [Q&A]
The response queries as shown in the docs don't seem exist as far as I can tell. 

[Response Query](https://docs.usebruno.com/scripting/response-query.html)

If you try to do this, it just says that 'res' is not a function.

**Comment by chrisramakers** [comment]
Have the same issue, seems like the docs aren't correct or it simply doesn't work anymore. There is a simple workaround though since the lodash library comes built in.

```
var _ = require('lodash');
console.log(_.get(res.getBody(), 'results[1].Name'))
```
